rasa/nlu/training_data/training_data.py

```diff
@@ -125,22 +182,78 @@ def sort_regex_features(self) -> None:
             self.regex_features, key=lambda e: "{}+{}".format(e["name"], e["pattern"])
         )
 
-    def as_json(self, **kwargs: Any) -> Text:
```

Comment:
for compatibility reasons I'd keep this as a deprecated function return nlu_as_json